docs/source/peer_ledger_snapshot.md

```diff
@@ -0,0 +1,163 @@
+# Taking ledger snapshots and using them to join channels
+
+For a peer to process transactions on a channel, it must endorse and validate transactions consistent with the other peers on a channel. This includes the "world state", maintained in the state database, which represents the current value of all of the keys on the ledger (who owns a particular asset, for example) as of the most recently committed block. There are two ways for a peer to get a copy of the necessary ledger data.
```

Comment:
The transition from the first to the second sentence got screwed up in the last changes. Probably my fault in the suggestions. @joealewine can you improve upon the wording?